### PR TITLE
Extended and reworked models, dockerfile cleanup

### LINKED ISSUE #42: [Database] Add missing models
There are a few models from our model missing, These should be implemented in the models.py in the backend.

- [ ] StationOccupancy
- [ ] CaregiversStation
- [ ] PatientJourney

### PR DESCRIPTION
**Pull request by jensUhl**
closes #42 .

Added the missing data models. Reworked alot of the data models and adapted `handleQuestions.py`. Cleaned up `Dockerfile` (only cosmetics).

**Added following data models:**

- PatientTransfers (was PatientJourney before)
- StationOccupancy
- StationWorkloadDaily (was CaregiversStation before)
- StationWorkloadMonthly (stores monthly averages of 

**Changes made to data models:**

- `Patient`: added current station, added dates for admission, discharged and deceased
- `PatientJourney`: renamed to PatientTransfers (in medical context, patient journey usually describes the analysis and development of a patients illness, not his location)
- `CaregiversStation`: renamed to StationWorkloadDaily. Station Workload has to be tracked daily, the monthly averages are calculated based on the daily entries and stored in extra table (StationWorkloadMonthly)

Added detailed comments for data fields in `models.py` and how they are calculated.
Reworked table descriptions.

`StationOccupancy` will be filled with data calculated from `Patient` and

### CODE REVIEW

**Review by lennart-huesing — CHANGES_REQUESTED** [review]
Throws many errors on startup since naming is wrong:
medical-staff-assessment-web-1  | ERRORS:
medical-staff-assessment-web-1  | backend.DailyClassification: (models.E012) 'unique_together' refers to the nonexistent field 'classification_date'.
medical-staff-assessment-web-1  | SystemCheckError: System check identified some issues:

File: backend/models.py [hunk 1]
```diff
@@ -3,50 +3,51 @@
...
 
     id = models.IntegerField(primary_key=True)
-    name = models.CharField(max_length=200)  # The name of the CareServiceOption
+    name = models.CharField(max_length=200)     # Concatenation of all columns, e.g. 'A-koerperpflege-1-1'
     field = models.ForeignKey('CareServiceField', on_delete=models.CASCADE)
     category = models.ForeignKey('CareServiceCategory', on_delete=models.CASCADE)
-    severity = models.IntegerField()  # 1-4 the degree of help need, higher means more help
-    list_index = models.IntegerField()  # Index of the questions within the filed, category and severity list
-    description = models.TextField()  # Content of the question
+    severity = models.IntegerField()            # Degree of needed help, increasing from 1 to 4
```
Comment (lennart-huesing) [inline backend/models.py:33]: Indent

File: backend/models.py [hunk 2]
```diff
@@ -3,50 +3,51 @@
 
 
 class CareServiceField(models.Model):
-    """The Field for CareServices often abbreviated with a short name like A or S"""
+    """General fields of care services, abbreviated with e.g. A or S."""
 
     id = models.IntegerField(primary_key=True)
-    name = models.CharField(max_length=128)  # For example 'Allgemeine Pflege'
-    short = models.CharField(max_length=8)  # For example 'A'
+    name = models.CharField(max_length=128)     # Field name, e.g. 'Allgemeine Pflege'
```
Comment (lennart-huesing) [inline backend/models.py:9]: Indent

**Review by lennart-huesing — CHANGES_REQUESTED** [review]
https://www.flake8rules.com/rules/E261.html
Two spaces should be before the inline comment

**Review by benCoder01 — APPROVED** [review]
Looks good to me

**Review by lennart-huesing — APPROVED** [review]

**Pull request merged**